quickstarts/microsoft.storage/storage-static-website/scripts/enable-static-website.ps1

```diff
@@ -0,0 +1,11 @@
+param(
+    [string] $ResourceGroupName,
+    [string] $StorageAccountName,
+    [string] $IndexDocument,
+    [string] $ErrorDocument404Path)
+
+$ErrorActionPreference = 'Stop'
+$storageAccount = Get-AzStorageAccount -ResourceGroupName $ResourceGroupName -AccountName $StorageAccountName
+
+$ctx = $storageAccount.Context
+Enable-AzStorageStaticWebsite -Context $ctx -IndexDocument $IndexDocument -ErrorDocument404Path $ErrorDocument404Path
```

Comment:
how do the files get content and on the server/storageAccount?

Reply:
In the original sample they don't. However, on reflection it probably makes sense to include the content, so I've revised the script to include this.